AudioStreaming/Streaming/AudioPlayer/AudioPlayer.swift

```diff
@@ -70,7 +70,10 @@ public final class AudioPlayer {
         playerContext.entriesLock.lock()
         let playingEntry = playerContext.audioPlayingEntry
         playerContext.entriesLock.unlock()
-        guard let entry = playingEntry, !entry.seekRequest.requested else { return 0 }
+        guard let entry = playingEntry else { return 0 }
+        if entry.seekRequest.requested {
+            return entry.seekRequest.time
+        }
```

Comment:
This is fine, although testing this with the example progress, the `if clause` will only get hit when the player is paused. But can't think of all the possible cases that might happen.

Although, I would wrap the if clause in SeekRequest's lock to be on the safe side
```suggestion
        entry.seekRequest.lock.lock()
        let seekRequested = entry.seekRequest.requested
        let seekTime = entry.seekRequest.time
        entry.seekRequest.lock.unlock()
        if seekRequested {
            return seekTime
        }
```

Reply:
Yes, it is only happens when the player is paused.
Thanks for the update.